src/components/Carousel/Carousel.tsx

```diff
@@ -25,19 +25,21 @@ const ButtonContainer: StyledComponent<
   position: relative;
 `;
 
-const ItemContainer: StyledComponent<
-  DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
-  {},
-  TockTheme
-> = styled.div`
+const ItemContainer: StyledComponent<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
+    {justifyContent?: String},
```

Comment:
```suggestion
    {},
```

This prop is no longer used